index.html

```diff
@@ -21,34 +21,34 @@ <h1>Experience the rush today</h1>
         </section>
         <section class = "features-section">
             <div class = "container">
-                <div class="feature">
-                    <div class="feature__icon_container">
+                <dl class="feature">
+                    <dt class="feature__icon_container">

```

Comment:
>The HTML <dt> element specifies a term in a description or definition list

I don't think `dt` is needed for image, as there is no term. So just use `div`

Reply:
Ok